frontend/src/modules/Notifications/index.js

```diff
@@ -0,0 +1,10 @@
+import { getModuleActiveStatus, ModuleNames } from 'utils';
+
+export const NotificationsModule = {
+  moduleDefinition: true,
+  name: 'notifications',
+  isEnvironmentModule: false,
+  resolve_dependency: () => {
+    return getModuleActiveStatus(ModuleNames.DATASETS);
```

Comment:
Is it a chat feature? If so, it can be used in datapipelines and dashboards. Checked it in the backend. If the backend is outdated and it's used only in datatset we could create a ticket to remove it from there too

Reply:
Notifications are not related with the Feed and Votes. Notifications are related to share notifications. They are the bell in the top right corner with the popover. I agree that we should assess if we should move the related views that the Notifications view stays in notifications instead of in design components popovers. This is something to review with @itsmo-amzn and @maryamkhidir